devops/scripts/create-sycl-user.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+
+set -e
+
+if [[ $# -eq 0 ]]; then
+  # When launched without arguments, we assume that it was launched as part of
+  # CI workflow and therefore a different kind of user is created
+  USER_NAME=sycl_ci
+  SET_PASSWD=true
+
+  # By default Ubuntu sets an arbitrary UID value, that is different from host
+  # system. When CI passes default UID value of 1001, some of LLVM tools fail to
+  # discover user home directory and fail a few LIT tests. Fixes UID and GID to
+  # 1001, that is used as default by GitHub Actions.
+  USER_ID=1001
+else
+  if [[ "${1:-}" != "--regular" ]]; then
+    echo "The only supported argument is --regular!"
+    exit 1
+  fi
+  USER_NAME=sycl
+  SET_PASSWD=false
+
+  # Some user id which is different from the one assigned to sycl_ci user
+  USER_ID=1234
```

Comment:
also sorry, you might have to rebase because i added a 24.04 oneapi docker image today

Reply:
No problem, see be4e0c6e040132bd5e8865c53dab6385e860ed4b for merge conflicts resolution and ff333a136e1baaba5c4bd9c7be789c08ecd58b71 for changes to that new dockerfile